tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -930,3 +925,128 @@ def get_volume_alert_counts():
                                          'alert_count': 0
                                          }
     return alert_counts
+
+
+def get_device_tree():
+    """{
```

Comment:
Is this sample output from the function?

Reply:
I feel you didnt get the point I meant. I wanted to explain the logic how the ancestors list is figured out.